docs/cplb.md

```diff
@@ -46,19 +52,23 @@ following:
   These do not provide any sort of security against ill-intentioned attacks, they are
   safety features to prevent accidental conflicts between VRRP instances in the same
   network segment.
+* If `VirtualServers` are used, the cluster configuration doesn't specify a non-empty
+  [`spec.api.externalAddress`][specapi]. `VRRPInstances` are compatible.
```

Comment:
hmm, I don't really get what this means. Could you rephrase this a bit

Reply:
I rephrased it, please review it again, I think it's better now but I'm not quite convinced to be honest